versions/2.0.0/asyncapi.md

```diff
@@ -92,6 +92,9 @@ A channel is an addressable component, made available by the server, for the org
 #### <a name="definitionsProtocol"></a>Protocol
 A protocol is the mechanism (wireline protocol OR API) by which [messages](#definitionsMessage) are exchanged between the application and the [channel](#definitionsChannel). Example protocol include, but are not limited to, AMQP, HTTP, JMS, Kafka, MQTT, STOMP, WebSocket.  
 
+#### <a name="definitionsBindings"></a>Bindings
+An AsyncAPI binding (or "protocol binding") is a mechanism to define protocol-specific information. Therefore, infrastructure details, deployment information, and everything that's not related to the protocol MUST be left apart.
+
```

Comment:
`deployment information,` No need for the comma after `information` if the summation is going to end.

`not related` I have a personal preference to use negations that are self-contained instead of adding the `not` so maybe `unrelated` is worth considering.

`MUST be left apart` maybe `MUST NOT be defined in a protocol binding` would be better here. I'm guessing that only protocol-specific information should be added here. Nothing else.

If I'm right and what's intended is `MUST NOT be defined [...]` then, you could consider dropping the double negation `not related` and `MUST NOT`. By changing it to a positive statement such as: `Therefore, a protocol binding MUST only define protocol-specific information.`. After the statement you can add examples of valid information to add. Since the range of valid definitions is probably much smaller than that of invalid definitions. 

This might seem redundant because the first sentence already mentioned `protocol-specific information`. However, that sentence introduced the concept of a `protocol binding` into the specification. So you can mention it again in the next one without to much of a problem.

Reply:
If I got it right, you suggest this without examples:

```
An AsyncAPI binding (or "protocol binding") is a mechanism to define protocol-specific information. Therefore, infrastructure details, deployment information and everything unrelated to the protocol MUST NOT be defined in a binding.
```


or this but with examples:

```
An AsyncAPI binding (or "protocol binding") is a mechanism to define protocol-specific information. Therefore, a binding MUST define only protocol-specific information.
```

GraphQL introduced a counter examples in their spec https://graphql.github.io/graphql-spec/June2018/ to show bad examples and I think here it would also make sense. I would say I'm with option 2, with examples, positive and negative

Reply:
@derberg Yes, if `MUST be left apart` can be interpreted as `MUST NOT be defined in a protocol binding.`
Which would indeed turn my suggestion into:
```
An AsyncAPI binding (or "protocol binding") is a mechanism to define protocol-specific information. Therefore, infrastructure details, deployment information and everything unrelated to the protocol MUST NOT be defined in a binding.
```
However, I would remove the double negation for clarity and cognitive load. By changing it to:
```
An AsyncAPI binding (or "protocol binding") is a mechanism to define protocol-specific information. Therefore, a protocol binding MUST only define protocol-specific information.
```
This makes it clear that only protocol-specific  information is acceptable. Since people tend to remember what was said at the start and the end of sentence. You could also state:
```
An AsyncAPI binding (or "protocol binding") is a mechanism to define protocol-specific information. Therefore, a protocol binding MUST define protocol-specific information only.
```
Afterwards you could add examples of valid information. Optionally you could also add examples of invalid information such as topology. Though that might be redundant because the sentence already clearly states that only protocol specific information is allowed. An example would be:

```
An AsyncAPI binding (or "protocol binding") is a mechanism to define protocol-specific information. Therefore, a protocol binding MUST define protocol-specific information only. A protocol binding MUST NOT contain information on topology, infrastructure, etc.. etc.. (this list is probably quite long)
```

Reply:
oh, I thought that by examples you actually mean an example yaml/json of binding information that is correct and that is incorrect. That would be my preference, but I do not have a strong opinion about it

Grammarly tells me that it is better to write `MUST define protocol-specific information only` rather than `MUST only define protocol-specific information` :) so I'm with the first one

Reply:
@derberg 
> I thought that by examples you actually mean an example yaml/json of binding information that is correct and that is incorrect.

That's an option as well. However, I meant the examples of invalid information which are given in the PR.

> Grammarly tells me that it is better to write [...]
Lets go with grammarly's suggestion then.
Which would give us this one:
```
An AsyncAPI binding (or "protocol binding") is a mechanism to define protocol-specific information. Therefore, a protocol binding MUST define protocol-specific information only. A protocol binding MUST NOT contain information on topology, infrastructure, etc.. etc.. (this list is probably quite long)
```
or this one:
```
An AsyncAPI binding (or "protocol binding") is a mechanism to define protocol-specific information. Therefore, a protocol binding MUST define protocol-specific information only. 
```
My preference would be the later. Since that one doesn't mention topology at all. So there is no reason to assume that it is valid information.

Fully-fledged yaml examples, both valid and in valid, can always be added later I presume.